Packs/Kafka/Integrations/KafkaV3/README.md

```diff
@@ -0,0 +1,177 @@
+Use the Kafka integration to manage messages and partitions.
+
+This integration was integrated and tested with version 2.7 of Kafka.
+
+This integration is fully compatible with Kafka_V2 configurations.
+
+Configure Kafka v3 on Cortex XSOAR
```

Comment:
lets generate new docs instead of using the v2 docs as the template changed a bit